modules/backend/classes/BackendController.php

```diff
@@ -49,6 +49,26 @@ class BackendController extends ControllerBase
      */
     public function __construct()
     {
+        // Find requested controller to determine if any middleware has been attached
+        $pathParts = explode('/', str_replace(Request::root() . '/', '', Request::url()));
+        if ($pathParts >= 2) {
+            // Drop off preceding "backend" URL part
+            array_shift($pathParts);
+            $path = implode('/', $pathParts);
```

Comment:
Does the above work correctly when the `backendUrl` is set to an empty string? (Usually for sites that don't use the CMS module, purely backend based)

Reply:
I didn't realise that was possible. I doubt it will at this stage but will make allowances for that.

Reply:
Sure is possible, required a little bit of tweaking here and there to make it seamless but it works great for building dedicated web applications powered by OctoberCMS

Reply:
All good, managed to work it out - needed to remove the CMS module from Composer and `config/cms.php`. :)